src/connections/DelayedJob.ts

```diff
@@ -0,0 +1,76 @@
+import { useEffect } from "react";
+import { createSelector } from "reselect";
+
+import { bulkUpdateJobs, listDelayedJobs } from "@/generated/v3/jobService/jobServiceComponents";
+import { bulkUpdateJobsFetchFailed, bulkUpdateJobsIsFetching } from "@/generated/v3/jobService/jobServicePredicates";
+import { DelayedJobData, DelayedJobDto } from "@/generated/v3/jobService/jobServiceSchemas";
+import { useConnection } from "@/hooks/useConnection";
+import { ApiDataStore } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+
+type DelayedJobCombinedConnection = {
+  delayedJobs?: DelayedJobDto[] | undefined;
+  delayedJobsIsLoading: boolean;
+  delayedJobsHasFailed: boolean;
+  bulkUpdateJobsIsLoading: boolean;
+  bulkUpdateJobsHasFailed: boolean;
+  updatedJobsResponse?: DelayedJobDto[];
+};
+
+const delayedJobsSelector = (store: ApiDataStore) =>
+  Object.values(store.delayedJobs ?? {}).map(resource => resource.attributes);
+
+const bulkUpdateJobsSelector = (store: ApiDataStore) => ({
+  bulkUpdateJobsIsLoading: bulkUpdateJobsIsFetching(store),
+  bulkUpdateJobsHasFailed: bulkUpdateJobsFetchFailed(store) != null,
+  updatedJobsResponse: Object.values(store.delayedJobs ?? {}).map(resource => resource.attributes as DelayedJobDto)
+});
```

Comment:
These two selectors are problematic because they're inputs to `createSelector`, and create a new object every time they're called. If their inputs don't change, they should return the same object every time to gain the true benefit of reselect. 

A better approach is for the delayedJobsSelector to be simpler: 
```
const delayedJobsSelector = (store: ApiDataStore) => store.delayedJobs
```

and to remove this bulk update selecot, instead relying directly on the isFetching / isFailed selectors in the `createSelector` call. So it becomes something more like:
```
const combinedSelector = createSelector([
  delayedJobsSelector, bulkUpdateJobsIsFetching, bulkUpdateJobsFetchFailed
], (delayedJobs, bulkUpdateJobsIsLoading, bulkUpdateJobsFailure) => ({
  delayedJobs: Object.values(delayedJobs ?? {}).map(resource => resource.attributes),
  ... // other members of the connection
}));
```

It's important for the members of the first argument array to `createSelector` to only return a unique object result if the inputs have changed, whenever possible.

Reply:
changed it and is working still.